test/integration-serial/serial_deploy_test.rb

```diff
@@ -603,6 +603,24 @@ def test_global_deploy_validation_catches_namespaced_cr
     wait_for_all_crd_deletion
   end
 
+  def test_global_deploy_prune_black_box_success
+    setup_template_dir("globals") do |target_dir|
```

Comment:
Let's augment this fixture set with one of the blacklisted kinds to show it doesn't get pruned.